### DISCUSSION TITLE
November 2024: The 'Three Years Off Dead For Tax Reasons' release

### DISCUSSION THREAD

**Discussion by denny** [denny/ShinyCMS-ruby#1823] [General]
Sorry for the unexpected hiatus in releases. In my defence, I nearly died shortly after the last one.

TL;DR, I turn out to have a massive arteriovenous malformation very near my brain stem, which ruptured on 18 June 2021, causing a haemorrhagic stroke that nearly killed me. In hindsight, that probably explains the insomnia and headaches between 2019 and 2021... I thought they were 'just' work stress.

Anyway, I have been busy learning how to talk again, and then walk again, and then recovering from six months of hospital-food-induced starvation (mostly by eating properly once I was home again, but some exercise too) - so all-in-all typing and coding have had to wait their turn. I'm getting there.

I don't think I've added or removed any features in this release, but there have been many many MANY gem updates - a lot of which were languishing in the dependabot queue while I recovered, and then more that flooded in once I cleared the initial backlog :)

<hr /><em>This discussion was created from the release <a href='https://github.com/denny/ShinyCMS-ruby/releases/tag/v24.11'>The 'Three Years Off Dead For Tax Reasons' Release</a>.</em>
